lib/feature/home/task_list/ui/task_list_view.dart

```diff
@@ -137,6 +141,18 @@ class TaskListView extends StatelessWidget {
     }
     return Center(child: Text(message));
   }
+
+  void _openCreateTask(BuildContext context) {
+    showCupertinoModalBottomSheet(
```

Comment:
Mind the back button here and make sure it plays well with the router delegate

Reply:
I checked this, and the back button functions as it should, thanks :)